test/user-model.js

```diff
@@ -1,218 +1,218 @@
-            })
-            await Review.writeNew(user.id, product.id, 4, 'So good!')
-        })
-        describe('GET', function () {
-            it('api/users', async function () {
-                const response = await app.get('/api/users')
-                const users = response.body
-                expect(response.status).to.equal(200)
-                expect(users).to.be.ok
-                expect(users).to.be.an('array')
-            })
-            it('api/users/:id', async function () {
-                const jack = await User.create({
-                    email: 'jack@snacker.com',
-                    password: 'abc123',
-                })
-                const response = await app.get(`/api/users/${jack.id}`)
-                const user = response.body
-                expect(response.status).to.equal(200)
-                expect(user).to.be.ok
-                expect(user).to.be.an('object')
-            })
-        })
-        describe('POST', () => {
-            it('api/users', async () => {
-                const response = await app.post('/api/users').send({
-                    email: 'test@snacker.com',
-                    password: 'test_pw',
-                })
-                const newUser = response.body
-                expect(response.status).to.equal(201)
-                expect(newUser).to.be.ok
-                expect(newUser).to.be.an('object')
-            })
-        })
-        describe('DELETE', () => {
-            it('api/users/:id', async () => {
-                const tempUser = await User.create({
-                    email: 'test@snacker.com',
-                    password: 'test_pw',
-                })
-                const response = await app.delete(`/api/users/${tempUser.id}`)
-                expect(response.status).to.equal(204)
-                expect(await User.findByPk(tempUser.id)).to.not.be.ok
-            })
-        })
-        describe('PUT', () => {
-            it('api/users/:id', async () => {
-                const tempUser = await User.create({
-                    email: 'test@snacker.com',
-                    password: 'test_pw',
-                })
-                const response = await app
-                    .put(`/api/users/${tempUser.id}`)
-                    .send({
-                        email: 'changed@snacker.com',
-                        password: 'changed_pw',
-                    })
-                const { email } = response.body
-                expect(response.status).to.equal(200)
-                expect(email).to.equal('changed@snacker.com')
-            })
-        })
-    })
-})
+  describe('User Routes', function () {
+    beforeEach(async function () {
+      const user = await User.create({
+        email: 'rosie@snacker.com',
+        password: '123ert',
+      });
+      const product = await Product.create({
+        title: 'puff',
+        brand: 'stay-puft',
+        description: 'tasty',
+        price: 1.1,
+        country: 'usa',
```

Comment:
please use 'location' instead

Reply:
I thought we got rid of this field in the Model

Reply:
Yap, you are right